mono/mini/mini-runtime.c

```diff
@@ -2212,9 +2212,18 @@ mono_jit_free_method (MonoDomain *domain, MonoMethod *method)
 	gboolean destroy = TRUE;
 	GHashTableIter iter;
 	MonoJumpList *jlist;
+	MonoJitDomainInfo *info = domain_jit_info (domain);
 
 	g_assert (method->dynamic);
 
+#ifdef ENABLE_INTERPRETER
+	if (mono_use_interpreter) {
+		mono_domain_jit_code_hash_lock (domain);
+		mono_internal_hash_table_remove (&info->interp_code_hash, method);
+		mono_domain_jit_code_hash_unlock (domain);
+	}
```

Comment:
Shouldn't this free the InterpMethod structure as well ? Or mention that its domain mempool allocated.

Reply:
it's mempool allocated; comment added.